### DISCUSSION TITLE
OC 0.8.8 + 2013 / 2017 MacBook Pro SMBIOS

### DISCUSSION THREAD

**Discussion by MultimediaLucario** [MultimediaLucario/Lenovo-ThinkPad-T440s#27] [Announcements]
![repository-open-graph-template new](https://user-images.githubusercontent.com/72415505/193061418-2920edc3-48f6-4397-b9da-e88fe475b626.png)

#
## Updated to OpenCore Version 0.8.8! 💻 🖥

This release comes with **four** EFI folders, one that supports up to **macOS 11 Big Sur** and older which contains the 2013 13" MacBook Pro SMBIOS, one that supports **macOS 12 Monterey** which contains the **2017** 13" MacBook Pro SMBIOS, one that supports **macOS 13 Ventura**, and an EFI folder that is optimized for ThinkPads with an **Intel Core I5-4210U**! **Please note that the SMBIOS for Ventura is still in **BETA** so do not use this for your everyday productivity!** 

<hr /><em>This discussion was created from the release <a href='https://github.com/MultimediaLucario/Lenovo-ThinkPad-T440s/releases/tag/OC-0.8.8'>OC 0.8.8 + 2013 / 2017 MacBook Pro SMBIOS</a>.</em>

**Comment by BritoHTK** [comment]
Hello, I have a T440s Laptop (ThinkPad) - Type 20AQ NS/PF01RP1K, the problem I have is that my i7 processor does not recognize me, the processor that it recognizes is an i5
